src/routes/shelters.js

```diff
@@ -302,6 +302,53 @@ router.get('/:id/delete_review', async(req, res) => {
     return;
 });
 
+router.get('/:id/edit_review', async (req, res) => {
+  let shelterId = req.params.id;
+  let loggedIn = helpers.isAuthenticated(req);
```

Comment:
The isAuthenticated helper function is no longer defined on the main branch since it was only req.session.user. So feel free to define it again or change the function call to `req.session.user`